core/server/common/src/main/java/alluxio/master/journal/ufs/UfsJournalWriter.java

```diff
@@ -351,7 +351,9 @@ public synchronized void write(JournalEntry entry) throws IOException {
             mDataOutputStream);
       } catch (IOException e) {
         mRotateLogForNextWrite = true;
-        throw e;
+        throw new IOException(String
+            .format("Failed to write journal file %s. Please ensure there is enough space left: %s",
```

Comment:
I would not include the "Please ensure there is enough space left" because I think it can me more misleading than helpful.